app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -113,7 +112,7 @@ class SmokeTest {
             verifyShortcutNameField(defaultWebPage.title)
             clickAddShortcutButton()
             clickAddAutomaticallyButton()
-            // verifyShortcutIcon()
+            verifyShortcutIcon()

```

Comment:
I meant removing this line, not sure if uncommenting would work

Reply:
okay